api/src/main/java/org/pmiops/workbench/db/model/DemographicSurveyEnum.java

```diff
@@ -77,35 +74,6 @@
           .put(Disability.FALSE, (short) 2)
           .build();
 
-  private static final BiMap<IndustryRole, Short> CLIENT_TO_STORAGE_INDUSTRY_ROLE =
```

Comment:
Surprised to see this here - I thought we collected all of these in DbStorageEnums